examples/all-clusters-app/esp32/main/main.cpp

```diff
@@ -84,6 +84,10 @@ using namespace ::chip::DeviceLayer;
 
```

Comment:
```
#elif CONFIG_DEVICE_TYPE_ESP32_DEVKITC || CONFIG_DEVICE_TYPE_ESP32_C3_DEVKITM
```

Reply:
ESP32 DevKitC and ESP32C3 DevKitM could use different gpio pins for status led. For C3, we can use GPIO8  control  W2812 RGB-LED on the board. But it is different from normal LED. So I don't think using the same GPIO Pin for the two module is better.